alexthompson207/rancid-tomatillos
# Feature/style movie trailer

## Is this a fix or a feature?

This is a feature. 

## What is the change?

Move API calls to a separate file. Also style single movie info view, and move movie image and movie trailer around.

## What does it fix?

N/a.

## Where should the reviewer start?

Look at the `apiCalls.js` file and then the `App` and `MoveInfo` components.

## How should this be tested?

Pull down this branch with git fetch, then open the site locally with `npm start`, all functionality should work the same, different styling in single movie view.
